content/topics/git/version-control-and-scrum/_index.md

```diff
@@ -78,22 +98,22 @@ Items in the backlog represent the conceived work for the team over the next 2-3
 Before items are able to be moved into the "To Do" column, they need to be looked over by the team during the "Storytime" meeting to make sure that the objectives are clear.
 Various labeling/measuring can take place on these tickets, like adding the feature they're associated with, the estimated work hours to complete the feature, and more.
 
-### To Do
+### This Sprint
 
-Tickets in to-do are in the queue for the team's current sprint.
+Tickets in this sprint are in the queue for the team's current sprint.
 The team commits to completing a given amount of tickets in the coming sprint during the "Planning" meeting.
 
-### Developing
+### Today
 
-Tickets moved to developing are being actively worked on by the person assigned to them.
+Tickets moved to today are being actively worked on by the person assigned to them.
 
 #### Git Action
 
 Before any coding occurs, pull down the latest version from "origin/master". Then, after coding is finished, create a new branch with a name corresponding to your ticket. Commit, push to origin, and create a pull request in Github.
```

Comment:
We follow feature branching {{contentlink here}}. This basically means that:
- when a dev starts working on a new issue, they should make a new branch. The branch should have a sensible name
- if the dev is writing code, they should be committing and pushing code. They should push every day that they made progress! Please insist on this,you will meet resistence and you will need to stand your ground. 
- Once the developer has made some changes worth committing then they make a PR. 

PRs should:
- have meaningful names
- be as small as possible so that whoever is doing the review can do a good job

Reply:
@tspmod the content of this comment from Sheena still seems to be missin